### PR TITLE
fix: dark theme for listbox

### LINKED ISSUE #1307: Dark theme does not correctly style list boxes
## 🐛 Bug report

Dark theme does not correctly style list boxes

### Steps to Reproduce

1. Mount a field as per stand process
3. Set theme to out-of-box 'dark'
4. Observe mounted listbox 

### Expected behavior

List box style to show in darkl mode

### Actual behavior

Listbox styling appears as light mode until a selection is made, at which point the list box shows as dark mode

### Versions

- nebula.js: 4.2.3
- Platform: Windows 11
- Browser: Tested in Chrome, Edge & Firefox

### PR DESCRIPTION
**Pull request by Caele**
## Motivation

Fixes #1307 and adds a test covering dark theme listboxes

## Requirements checklist

- [ ] Api specification
  - [ ] Ran `yarn spec`
    - [ ] No changes **_OR_** API changes has been formally approved
- [ ] Unit/Component test coverage
- [ ] Correct PR title for the changes (fix, chore, feat)

When build and tests have passed:

- [ ] Add code reviewers, for example @qlik-oss/nebula-core

### CODE REVIEW

**Review by quanho — APPROVED** [review]

**Comment by quanho** [comment]
Look good to me!

**Review by johanlahti — APPROVED** [review]

File: apis/nucleus/src/components/listbox/assets/styling.js [hunk 1]
```diff
@@ -153,7 +154,7 @@ export default function getStyles({ app, themeApi, theme, components = [], check
     ? resolveBgImage({ bgImage: themeOverrides.background.image }, app)
     : undefined;
 
-  const searchBgColor = 'rgba(255, 255, 255, 0.7)';
+  const searchBgColor = themeType === 'light' ? 'rgba(255, 255, 255, 0.7)' : 'rgba(255, 255, 255, 0.2)';
```
Comment (T-Wizard) [inline apis/nucleus/src/components/listbox/assets/styling.js:157]: consider adding searchBgColor to the themes

Comment (Caele) [reply apis/nucleus/src/components/listbox/assets/styling.js:157]: yea, looked at that a bit now. Became messy. But I think using the normal bg color + an alpha might be better.

Comment (T-Wizard) [reply apis/nucleus/src/components/listbox/assets/styling.js]: OK

**Pull request merged**